### PR TITLE
[Tests] Add test for TopicCountEquallyDivideBundleSplitAlgorithm.

### PR DESCRIPTION
**Pull request by mattisonchao**
### Modifications

add test for TopicCountEquallyDivideBundleSplitAlgorithm.

### CODE REVIEW

File: pulsar-broker/src/test/java/org/apache/pulsar/common/naming/TopicCountEquallyDivideBundleSplitAlgorithmTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            Mockito.doReturn(mockNamespaceBundleFactory)
+                    .when(mockNamespaceBundle1).getNamespaceBundleFactory();
+            long hashValue = Hashing.crc32().hashString(topic, Charsets.UTF_8).padToLong();
+            Mockito.doReturn(hashValue)
+                    .when(mockNamespaceBundleFactory).getLongHashCode(topic);
+        });
+        Assert.assertEquals((long)algorithm.getSplitBoundary(mockNamespaceService1, mockNamespaceBundle1).join(),correctResult);
+    }
+
+    private long getCorrectResult(List<String> mockTopics){
```
Comment (lhotari) [inline pulsar-broker/src/test/java/org/apache/pulsar/common/naming/TopicCountEquallyDivideBundleSplitAlgorithmTest.java:73]: "getCorrectResult" is a very vague name for a method.
It seems that a better approach would be to inline this method.

Comment (mattisonchao) [reply pulsar-broker/src/test/java/org/apache/pulsar/common/naming/TopicCountEquallyDivideBundleSplitAlgorithmTest.java:73]: OK, thank you, I will fix it.  : )

**Comment by mattisonchao** [comment]
i've been fix it,could you review it ? @lhotari

**Comment by mattisonchao** [comment]
I've been fixing it, could you review it? @lhotari

**Review by lhotari — APPROVED** [review]
LGTM

**Comment by lhotari** [comment]
Btw. The prefix "Feat #" you have for the PR title doesn't follow the conventions used in Apache Pulsar. Please take a look at titles of other PRs.

File: pulsar-broker/src/test/java/org/apache/pulsar/common/naming/TopicCountEquallyDivideBundleSplitAlgorithmTest.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+ * under the License.
+ */
+package org.apache.pulsar.common.naming;
+
+
+import com.google.common.base.Charsets;
+import com.google.common.collect.Lists;
+import com.google.common.hash.Hashing;
+import org.apache.pulsar.broker.namespace.NamespaceService;
+import org.junit.Test;
```
Comment (lhotari) [inline pulsar-broker/src/test/java/org/apache/pulsar/common/naming/TopicCountEquallyDivideBundleSplitAlgorithmTest.java:26]: Invalid import in TestNG tests. Please use the TestNG Test annotation.

**Review by lhotari — CHANGES_REQUESTED** [review]

**Review by lhotari — CHANGES_REQUESTED** [review]
Remove import for org.junit.Test

File: pulsar-broker/src/test/java/org/apache/pulsar/common/naming/TopicCountEquallyDivideBundleSplitAlgorithmTest.java (hunk 2 above)
Comment (lhotari) [reply pulsar-broker/src/test/java/org/apache/pulsar/common/naming/TopicCountEquallyDivideBundleSplitAlgorithmTest.java:26]: I noticed some previous PRs have been merged with org.junit.Test annotations. That's fixed in #11020 so you don't have to worry about the imports in previous PRs.

Comment (mattisonchao) [reply pulsar-broker/src/test/java/org/apache/pulsar/common/naming/TopicCountEquallyDivideBundleSplitAlgorithmTest.java]: fixed :)

**Comment by mattisonchao** [comment]
Thank you for your rigorous review 👍🏼 , I've been fixing some problem. 
Could you review it again?

**Review by lhotari — APPROVED** [review]

**Review by merlimat — APPROVED** [review]

**Pull request merged**